MarkdownView/markdownview/src/main/AndroidManifest.xml

```diff
@@ -1,11 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="us.feras.mdv">
-
-    <application
-        android:allowBackup="true"
-        android:label="@string/app_name"
-        android:supportsRtl="true">
```

Comment:
One suggestion/improvement I would personally find really useful is the following. In MarkdownView.java on line 135 there is the following 2 lines:

    MarkdownProcessor m = new MarkdownProcessor();
    String html = m.markdown(txt);

I would like to have a chance to subclass MarkdownView and intercept the HTML String before it is loaded into the WebView. Hence a function like this:

    protected String markdownToHtmlString (String txt) {
        MarkdownProcessor m = new MarkdownProcessor();
        return m.markdown(txt);
    }

Other than that I haven't used the MarkdownView extensively yet, but this would be nice as the CMS I'm connecting to works some peculiar magic with the URLs for images.